packages/client/tsconfig.json

```diff
@@ -2,7 +2,8 @@
     "extends": "@streamr/dev-config/ts/tsconfig.browser.json",
     "compilerOptions": {
         "outDir": "dist",
-        "rootDirs": ["src", "test"]
+        "rootDirs": ["src", "test"],
+        "resolveJsonModule": true
```

Comment:
`resolveJsonModule` shouldn't be necessary, did you hit an issue? https://github.com/streamr-dev/network-monorepo/blob/193e6eac1e150cd7b3f1a20a7c86e814bcb12dc0/packages/dev-config/ts/tsconfig.browser.json#L22

Reply:
It's set in @streamr/dev-config: https://github.com/streamr-dev/network-monorepo/blob/main/packages/dev-config/ts/tsconfig.browser.json#L22

Reply:
I think tests import contract jsons directly

Reply:
I can delete it, too